### PR TITLE
Docs: Updated index.html

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

File: files/main.css [hunk 1]
```diff
@@ -257,7 +257,6 @@ h1 a {
 	#panel #language {
 		font-family: 'Roboto Mono', monospace;
 		font-size: 1rem;
-		line-height: 1rem;
```
Comment (Mugen87) [inline files/main.css:260]: Um, why is this line changed?

Comment (linbingquan) [reply files/main.css:260]: > Um, why is this line changed?

Sorry, I didn't compare the result. Here you are.

|  Before   | After  |
|  ----  | ----  |
| before_en  | after_en |
| before_zh  | after_zh |

Comment (Mugen87) [reply files/main.css:260]: Ah okay, so this is required because changing from `en` to `English` requires more space.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
This breaks the design in Safari a bit:

Screen Shot 2020-06-12 at 11 55 17 AM

Screen Shot 2020-06-12 at 11 54 12 AM

Chinese people don't like `zh`?

**Comment by linbingquan** [comment]
> This breaks the design in Safari a bit:

I am work in Windows 10 OS. The breaks is `e` is not capitalization? but I felt confused after saw this commit https://github.com/mrdoob/three.js/commit/06bbffed488f9705a00bc3437851340c67d8d25f.

https://github.com/mrdoob/three.js/blob/330d29d69f1d8414369af7dd63c5df2171e8a560/docs/index.html#L33

> Chinese people don't like `zh`?

It's not intuitive.

I saw the [React](https://reactjs.org/languages) and [Vue](https://vuejs.org/index.html) website translations also use native language.

**Comment by Mugen87** [comment]
TBH, I find the new `English` term misleading. It looks like it's already a search term.

**Comment by mrdoob** [comment]
How about "en" and "中文"? https://github.com/mrdoob/three.js/commit/1af3252b099dbf5a8043fa1c2daa05aac16e1eb9
